src/main/java/hill/weatherconfidence/ingest/ScheduledCollector.java

```diff
@@ -1,31 +1,52 @@
 package hill.weatherconfidence.ingest;
 
 import hill.weatherconfidence.IKmsDecrypter;
+import hill.weatherconfidence.ingest.model.GeocodeItem;
+import hill.weatherconfidence.ingest.model.GeocodeResponse;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
-import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
 import org.springframework.scheduling.annotation.Scheduled;
 import org.springframework.stereotype.Component;
-import org.springframework.web.reactive.function.client.WebClient;
+
+import java.util.ArrayList;
+import java.util.List;
 
 @Component
 public class ScheduledCollector {
 
     private static final Logger log = LoggerFactory.getLogger(ScheduledCollector.class);
-
     private final IKmsDecrypter kmsDecrypter;
+    private final String baseUrl;
 
-    private final WebClient webClient;
+    private final List<GeocodeItem> locations = new ArrayList<>();
 
-    public ScheduledCollector(IKmsDecrypter kmsDecrypter, WebClient webClient) {
+    public ScheduledCollector(IKmsDecrypter kmsDecrypter, @Value("${weatherApi.baseUrl}") String baseUrl) {
         this.kmsDecrypter = kmsDecrypter;
-        this.webClient = webClient;
+        this.baseUrl = baseUrl;
+    }
+
+    private void init() {
+        String apiKey = kmsDecrypter.getDecryptedApiKey();
+        GeocodeClient geocodeClient = new GeocodeClient(baseUrl, apiKey);
+        GeocodeResponse geocodeResponse = geocodeClient.geocode("Hemel Hempstead").block();
```

Comment:
This block() call is temporary.  This probably needs rethinking - at least if we start to handle other locations.